pages/ErrorScreen/index.tsx

```diff
@@ -0,0 +1,22 @@
+import clsx from 'clsx';
+import Image from 'next/image';
+import emMonocle from '../../public/assets/emojis/monocle.svg';
+import styles from './styles.module.css';
+
+export interface IErrorScreen {
+  className?: string;
+  message: string;
+  title: string;
+}
+
+function ErrorScreen({className, message, title}: IErrorScreen) {
```

Comment:
Is this a component or a page?
it is on the pages folder but it is being used as a component

Reply:
is componenttt